client/src/components/CreatePost/CreatePost.js

```diff
@@ -82,7 +83,7 @@ const Step2 = ({ user }) => {
 
 const Step3 = ({ onCancel }) => {
   const { currentStep, setCurrentStep, setPostId } = useContext(
-    CreatePostContext,
+    CreatePostContext
```

Comment:
no trailing comma?

Reply:
Yeah this is an issue when I apply prettier --write via docker & didn't catch this mistake. We have `trailingComma: 'all' in root level .prettier.yml. We've got to find a better way to make this work smoothly in docker - see https://github.com/FightPandemics/FightPandemics/issues/786

Reply:
Ohh interesting though! That explains a lot! @CeesWang is probably not running docker, which is why his code gets reformatted on commit and messes some of his stuff...